algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/util/FormatTest.kt

```diff
@@ -88,7 +88,10 @@ class FormatTest {
 
     @Test
     fun `Should format lab values with first letter capitalized measure display, formatted value and display unit`() {
-        assertEquals("Indirect bilirubin 4.0 umol/L", Format.labValue(LabMeasurement.INDIRECT_BILIRUBIN, 4.005, LabUnit.MICROMOLES_PER_LITER))
+        assertEquals(
+            "Indirect bilirubin 4.0 umol/L",
+            Format.labValue(LabMeasurement.INDIRECT_BILIRUBIN, 4.005, LabUnit.MICROMOLES_PER_LITER)
+        )
```

Comment:
We should ideally use `assertj` here, i.e.
```suggestion
        assertThat(Format.labValue(LabMeasurement.INDIRECT_BILIRUBIN, 4.005, LabUnit.MICROMOLES_PER_LITER))
            .isEqualTo("Indirect bilirubin 4.0 umol/L")
```